firefox-ios/Client/Frontend/Home/HomepageViewController.swift

```diff
@@ -384,7 +384,11 @@ class HomepageViewController:
 
     @objc
     private func dismissKeyboard() {
-        if currentTab?.lastKnownUrl?.absoluteString.hasPrefix("internal://") ?? false {
+        /* homepage and error page, both are "internal" url, making
+           topsites on homepage inaccessible from error page 
+           when address bar is selected hence using "about/home".
+        */
+        if currentTab?.lastKnownUrl?.absoluteString.contains(AboutHomeHandler.path) ?? false {
```

Comment:
This would result in problems for any url containing `about/home` anywhere in the url (e.g. https://google.com/about/home). It would be better to use `hasPrefix` again and compare against `\(InternalURL.baseUrl)/\(AboutHomeHandler.path)`.

```suggestion
        if currentTab?.lastKnownUrl?.absoluteString.hasPrefix("\(InternalURL.baseUrl)/\(AboutHomeHandler.path)") ?? false {
```

Reply:
Understood. I will do the said changes. Thank you!